### DISCUSSION TITLE
include_all_commits=true is not working correctly.

### DISCUSSION THREAD

**Discussion by yellowflash2041** [anuraghazra/github-readme-stats#2255] [Q&A] [answered]
### Describe the bug

I set the include_all_commits true in my github-stats.
But the total commits count is 0.
![image](https://user-images.githubusercontent.com/110062591/199789953-89ff8113-058d-4b89-8a5d-b9885d7c3672.png)

### Expected behaviour

![image](https://user-images.githubusercontent.com/110062591/199790075-aa36c003-b680-401a-ad1b-d652a6e2fffb.png)

### Screenshots / Live demo link

https://github.com/yellowflash2041

### Additional context

I fixed after setting include_all_commits before other properties.
![image](https://user-images.githubusercontent.com/110062591/199790345-cc058e3e-bcfa-4b3b-bfb5-03a3c42f1f9f.png)

**Comment by rickstaa** [comment, marked as answer]
@yellowflash2041 Thanks for your question. I checked your card, and it seems to work:

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=yellowflash2041&include_all_commits=true)](https://github.com/anuraghazra/github-readme-stats)
```

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=yellowflash2041&include_all_commits=true)](https://github.com/anuraghazra/github-readme-stats)

The issue you are experiencing is because of caching. We display 0 when GitHub Rest API has an error this result is then cached for 4 hours. Merging #2177 will fix this.

**Reply by yellowflash2041** [reply]
Thank you, @rickstaa
